### PR TITLE
Ensuring Appium times out gracefully if webkit does not return a response

### PR DESCRIPTION
**Pull request by ssynix**
Addressing #42.

I went with a simple 5 sec timeout. Let me know if it needs to be configured through a capability.

### CODE REVIEW

File: lib/remote-debugger.js [hunk 1]
```diff
@@ -19,6 +19,9 @@ const DEBUGGER_TYPES = {
 const SELECT_APP_RETRIES = 20;
 const REMOTE_DEBUGGER_PORT = 27753;
 
+/* How many milliseconds to wait for webkit to return a response before timing out */
+const RPC_RESPONSE_TIMEOUT_MS = 5000;
```
Comment (imurchie) [inline lib/remote-debugger.js]: If it is only used for Webkit, why not declare it there?

Comment (ssynix) [inline lib/remote-debugger.js]: Just so the constants are defined in the same place. Do you prefer I move it to the other file?

Comment (imurchie) [inline lib/remote-debugger.js]: No, I guess that makes some sense. :)

**Comment by imurchie** [comment]
Sorry, I lost track of this. Thanks for the work in getting this in!

**Comment by imurchie** [comment]
Published in `appium-remote-debugger@3.1.9`

**Comment by ssynix** [comment]
Thanks! Will this make it to `1.6.1`?

**Comment by imurchie** [comment]
It will. It should be in the beta.

**Pull request merged**